tomls/omnibus-arbitrum-sepolia/rewards/sBTC_perps_liquidations_rewards.toml

```diff
@@ -1,7 +1,7 @@
 [provision.RewardsDistributor_sBTC_liquidation_rewards]
 source = "synthetix-rewards-dist-ext:3.6.0"
 target = "synthetix-rewards-dist-ext:3.6.0@perps-sBTC"
-options.salt = "<%= settings.salt %>"
+options.salt = "<%= settings.salt %>-perps-sBTC"
```

Comment:
Looks like it was a false alarm. I checked and we did in fact deploy different contracts. The reason is that they do not use CREATE2 and are deployed separately. It is probably better to avoid this change.

https://usecannon.com/packages/synthetix-rewards-dist-ext/latest/421614-perps-sBTC
https://usecannon.com/packages/synthetix-rewards-dist-ext/latest/421614-perps-sETH

Sorry for the confusion

Reply:
alright reverting